test/badges.test.ts

```diff
@@ -202,18 +202,19 @@ describe('Proxy upgrades', () => {
     await mintBadge()
   })
 
-  it('Should upgrade the Badge contract then create raft/spec/badge', async () => {
-    // deploy contracts
-    const { badgesProxy } = deployed
-
-    const badgesV2 = await ethers.getContractFactory('BadgesV2')
-    const upgradedV2Contract = await upgrades.upgradeProxy(badgesProxy.address, badgesV2)
-    await upgradedV2Contract.deployed()
-
-    const v2 = await upgradedV2Contract.getVersion()
-    expect(v2).equal(2)
+  it.only('Should run an upgrade BadgesOld.sol to BadgesNew.sol', async () => {
+    const { raftProxy, owner, specDataHolderProxy } = deployed
+
+    const badgesOld = await ethers.getContractFactory('BadgesOld')
```

Comment:
Is is possible to make an easy script to regenerate the BadgedOld/New files? :)

Reply:
yes, I'm going to actually make a separate PR for that now.